browser/playlist/playlist_page_handler.cc

```diff
@@ -47,8 +57,9 @@ void PlaylistPageHandler::GetAllPlaylists(
     std::vector<mojo::StructPtr<playlist::mojom::PlaylistItem>> items;
     for (const auto& item : playlist.items) {
       items.push_back(playlist::mojom::PlaylistItem::New(
-          item.id, item.title, GURL(item.page_src), GURL(item.media_file_path),
-          GURL(item.thumbnail_path), item.media_file_cached));
+          item.id, item.title, GURL(item.page_src),
+          GURL("file://" + item.media_file_path),
+          GURL("file://" + item.thumbnail_path), item.media_file_cached));
```

Comment:
Sorry, but I think my new PR would revert this line: https://github.com/brave/brave-core/pull/16439/files#r1054289846

Reply:
Cool. reverting my change.